### PR TITLE
Poster: itk-elastix: Medical image registration in Python

### PR DESCRIPTION
**Pull request by ntatsisk**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*:  

*Reviewers*:

### CODE REVIEW

**Review by thewtex — APPROVED** [review]

File: presentations/posters/itk-elastix-poster/info.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        {
+            "name": "Viktor van der Valk",
+            "affiliation": "Division of Image Processing, Department of Radiology, Leiden University Medical Center, Leiden, the Netherlands",
+            "orcid": ""
+        },
+        {
+            "name": "Tom Birdsong",
+            "affiliation": "Medical Computing Group, Kitware, Inc, Carrboro, NC, USA",
+            "orcid": ""
```
Comment (thewtex) [inline presentations/posters/itk-elastix-poster/info.json]: @tbirdso 's ORCiD is 0009-0001-7007-684X

File: presentations/posters/itk-elastix-poster/info.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        {
+            "name": "Dženan Zukić",
+            "affiliation": "Medical Computing Group, Kitware, Inc, Carrboro, NC, USA",
+            "orcid": "0000-0002-9546-9703"
```
Comment (dzenanz) [inline presentations/posters/itk-elastix-poster/info.json:26]: This is correct.

File: presentations/posters/itk-elastix-poster/info.json (hunk 1 above)
Comment (thewtex) [reply presentations/posters/itk-elastix-poster/info.json]: Added in 516d1d3

**Review by dzenanz — APPROVED** [review]

**Review by tbirdso — APPROVED** [review]
Thank you!

**Review by deniederhut — APPROVED** [review]
giphy-2999900406

**Pull request merged**